tc/sgx/trusted_worker_manager/enclave/work_order_processor_wpe.cpp

```diff
@@ -0,0 +1,289 @@
+            wo_data.Unpack(enclave_data, data_object, WPE_ENCLAVE,
+                wo_pre_proc_keys.out_data_keys[i].decrypted_data);
+            data_items_out.emplace_back(wo_data);
+        }
+    }  // WorkOrderProcessorWPE::ParseJsonInput
+
+    ByteArray WorkOrderProcessorWPE::CreateJsonOutput() {
+        JSON_Status jret;
+
+        // Create the response structure
+        JsonValue resp_value(json_value_init_object());
+        tcf::error::ThrowIf<tcf::error::RuntimeError>(
+            !resp_value.value, "Failed to create the response object");
+
+        JSON_Object* resp = json_value_get_object(resp_value);
+        tcf::error::ThrowIfNull(resp,
+            "Failed on retrieval of response object value");
+
+        JsonSetStr(resp, "jsonrpc", "2.0", "failed to serialize jsonrpc");
+        JsonSetNumber(resp, "id", json_request_id,
+            "failed to serialize json request id");
+
+        jret = json_object_set_value(resp, "result", json_value_init_object());
+        tcf::error::ThrowIf<tcf::error::RuntimeError>(
+            jret != JSONSuccess, "failed to serialize result");
+
+        JSON_Object* result = json_object_get_object(resp, "result");
+        tcf::error::ThrowIfNull(result, "failed to serialize the result");
+
+        JsonSetStr(result, "workOrderId", work_order_id.c_str(),
+            "failed to serialize work order id");
+        JsonSetStr(result, "workloadId", workload_id.c_str(),
+            "failed to serialize workload id");
+        JsonSetStr(result, "workerId", worker_id.c_str(),
+            "failed to serialize worker id");
+        JsonSetStr(result, "requesterId", requester_id.c_str(),
+            "failed to serialize requester id");
+        JsonSetStr(result, "workerNonce", worker_nonce.c_str(),
+            "failed to serialize worker nonce");
+        JsonSetStr(result, "workerSignature", worker_signature.c_str(),
+            "failed to serialize worker signature");
+        // Pack verification key and verification key signature
+        // from preprocessed json
+        JsonSetStr(result, "verificationKey",
+            ByteArrayToBase64EncodedString(
+                wo_pre_proc_keys.verification_key).c_str(),
+            "failed to serialize verification key");
+        JsonSetStr(result, "verificationKeySignature",
+            ByteArrayToBase64EncodedString(
+                wo_pre_proc_keys.verification_key_signature).c_str(),
+            "failed to serialize verification key signature");
+
+        jret = json_object_set_value(result, "outData", json_value_init_array());
+        tcf::error::ThrowIf<tcf::error::RuntimeError>(
+            jret != JSONSuccess, "failed to serialize the result data");
+
+        JSON_Array* data_array = json_object_get_array(result, "outData");
+        tcf::error::ThrowIfNull(data_array,
+            "failed to serialize the dependency array");
+
+        for (auto out_data : data_items_out)
+            out_data.Pack(data_array);
```

Comment:
1) Please call the base class `WorkOrderProcessor::createJsonOutput()`
2) Add extVerificationKey and extVerificationKeySignature in overrideen function
3) Call base class `WorkOrderProcessor::serializeJsonOutput()`

Reply:
Done